### PR TITLE
Fix a sentence in contributing to qiskit docs

### PR DESCRIPTION
**Pull request by SooluThomas**
### Summary

Issue was found by a translator

### CODE REVIEW

File: docs/contributing_to_qiskit.rst [hunk 1]
```diff
@@ -78,8 +78,6 @@ Development Environment and install the appropriate package **from source**.
 For a quick guide on how to do this for qiskit-terra take a look at the
 `How to Install Qiskit - Contributors <https://www.youtube.com/watch?v=Pix2MFCtiOo>`__ YouTube video.
 
-You can learn how to install different Qiskit packages from source in the :ref:`install-from-source`.
-
```
Comment (SooluThomas) [inline docs/contributing_to_qiskit.rst:82]: We don't have `install-from-source` section anymore to add a reference link anymore.

File: docs/qc_intro.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ of the quantum computation. Contrast this with **real-time computation**, where
 computation occurs within the decoherence time of the quantum device.
 
 Constructing complex quantum circuits with minimal effort is at the heart of Qiskit,
-that support a rich feature set of operations, and can be passed to a range of
+which supports a rich feature set of operations and can send your circuits to a range of
 :ref:`quantum computers <qc-intro-computers>` or classical simulators.
```
Comment (SooluThomas) [inline docs/qc_intro.rst:162]: Hopefully I understand the statement correctly. Here "a rich feature set of operations" is supported by "Qiskit" and it is the "quantum circuits" that "can be passed to a range of quantum computers and classical simulators". Please correct me if I'm wrong.

File: docs/contributing_to_qiskit.rst [hunk 3]
```diff
@@ -171,7 +169,7 @@ merged in a timely manner.
 - You have followed the requirements in the CONTRIBUTING.md file for the specific repo you are
   contributing to.
 - All CI checks pass (it's recommended to run tests and lint checks locally before pushing).
-- New tests have for any new functionality that has been introduced.
+- New tests have to be added for any new functionality that has been introduced.
```
Comment (kevinhartman) [inline docs/contributing_to_qiskit.rst]:
```suggestion
- New tests have been added for any new functionality that has been introduced.
```

... to make the wording a little more checklist-y.

File: docs/qc_intro.rst (hunk 2 above)
Comment (kevinhartman) [reply docs/qc_intro.rst:162]: IMO your correction is fine 🙂.

**Comment by kevinhartman** [comment]
You'll need to merge `master` to this PR to get the docs tests passing, I believe.

**Review by kevinhartman — APPROVED** [review]
LGTM

**Pull request merged**